node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/NodeState.java

```diff
@@ -9,5 +9,5 @@
  * @author freva
  */
 public enum NodeState {
-    provisioned, ready, reserved, active, inactive, dirty, failed, parked
+    provisioned, ready, reserved, active, inactive, dirty, failed, parked, deprovisioned
```

Comment:
Should also be added to `com.yahoo.vespa.hosted.controller.api.integration.noderepository.NodeState`.

Reply:
Done - thanks